src/App.js

```diff
@@ -2,15 +2,51 @@ import React from 'react';
 import './App.scss';
 
 class App extends React.Component {
-  addOne = () => {};
+  state = {
+    count: 0,
+  };
 
-  add100 = () => {};
+  addOne = () => {
+    this.setState(({ count }) => ({
+      count: count + 1,
+    }));
+  };
 
-  increase = () => {};
+  add100 = () => {
+    this.setState(({ count }) => ({
+      count: count + 100,
+    }));
+  };
+
+  increase = () => {
+    this.setState(({ count }) => {
+      this.addOne();
+
+      if (count % 5 === 0 && count !== 0) {
+        this.add100();
+      }
+    });
+  };
```

Comment:
```suggestion
  increase = () => {
    this.addOne();

    if (count % 5 === 0 && count !== 0) {
      this.add100();
    }
  };
```
No need in setState here. You call two functions that already call setState method inside them

Reply:
Хотел деструктуризацию сделать, по другому не получалось